os/net/ipv6/uip-icmp6.c

```diff
@@ -253,10 +253,6 @@ uip_icmp6_error_output(uint8_t type, uint8_t code, uint32_t param) {
 void
 uip_icmp6_send(const uip_ipaddr_t *dest, int type, int code, int payload_len)
 {
-  LOG_INFO("Sending ICMPv6 packet to ");
-  LOG_INFO_6ADDR(&UIP_IP_BUF->destipaddr);
```

Comment:
Why don't you replace ` &UIP_IP_BUF->destipaddr` with `dest` here instead of moving the three lines to the bottom?

Reply:
There is no strong reason for that. Passing `dest` is ok for me, too.

However, I think printing `UIP_IP_BUF->destipaddr` (after being updated) is more true to this function's behavior observed from outside. Correctness of printing `dest` depends on the fact that the function copies it to `UIP_IP_BUF->destipaddr`.